data/pc/1.19.4/protocol.json

```diff
@@ -0,0 +1,6540 @@
+      ]
+    ],
+    "particleData": [
+      "switch",
+      {
+        "compareTo": "$compareTo",
+        "fields": {
+          "2": [
+            "container",
+            [
+              {
+                "name": "blockState",
+                "type": "varint"
+              }
+            ]
+          ],
+          "3": [
+            "container",
+            [
+              {
+                "name": "blockState",
+                "type": "varint"
+              }
+            ]
+          ],
+          "14": [
+            "container",
+            [
+              {
+                "name": "red",
+                "type": "f32"
+              },
+              {
+                "name": "green",
+                "type": "f32"
+              },
+              {
+                "name": "blue",
+                "type": "f32"
+              },
+              {
+                "name": "scale",
+                "type": "f32"
+              }
+            ]
+          ],
+          "15": [
+            "container",
+            [
+              {
+                "name": "fromRed",
+                "type": "f32"
+              },
+              {
+                "name": "fromGreen",
+                "type": "f32"
+              },
+              {
+                "name": "fromBlue",
+                "type": "f32"
+              },
+              {
+                "name": "scale",
+                "type": "f32"
+              },
+              {
+                "name": "toRed",
+                "type": "f32"
+              },
+              {
+                "name": "toGreen",
+                "type": "f32"
+              },
+              {
+                "name": "toBlue",
+                "type": "f32"
+              }
+            ]
+          ],
+          "24": [
+            "container",
+            [
+              {
+                "name": "blockState",
+                "type": "varint"
+              }
+            ]
+          ],
+          "35": [
+            "container",
+            [
+              {
+                "name": "item",
+                "type": "slot"
+              }
+            ]
+          ],
+          "36": [
+            "container",
+            [
+              {
+                "name": "origin",
+                "type": "position"
+              },
+              {
+                "name": "positionType",
+                "type": "string"
+              },
+              {
+                "name": "destination",
+                "type": [
+                  "switch",
+                  {
+                    "compareTo": "positionType",
+                    "fields": {
+                      "minecraft:block": "position",
+                      "minecraft:entity": "varint"
+                    }
+                  }
+                ]
+              },
+              {
+                "name": "ticks",
+                "type": "varint"
+              }
+            ]
+          ]
```

Comment:
Missing the ID 95

![image](https://github.com/PrismarineJS/minecraft-data/assets/6418522/a848e7b3-5113-46fb-8425-4c5dcf927818)

Reply:
Valid here and on 1.19.3 also (and maybe older versions)